contrib/fwupd.spec.in

```diff
@@ -217,6 +217,7 @@ Summary: fwupd wrapper for Qubes OS - dom0 scripts
 Requires:   gcab
 Requires:   fwupd >= 1.5.7
 Requires:   libjcat >= 0.1.6
+Requires:   python3-packaging
```

Comment:
I think that would upset the CoreOS people -- do we really need it for all cases?

Reply:
It is imported in qubes scripts, so yes, it is needed there. But note it's added only to qubes subpackage, not the main one.

Reply:
AHH, gotcha, thanks.